common/src/main/java/com/wynntils/core/webapi/WebManager.java

```diff
@@ -76,19 +76,21 @@ public final class WebManager {
 
     private static List<MapProfile> maps = null;
 
-    private static String userAgent;
+    private static final String USER_AGENT;
```

Comment:
Can't you do like: 
```
    private static final String USER_AGENT = String.format(...);
```
?

Reply:
static clinit blocks are the worst...

Reply:
During testing yesterday VERSION was still an empty string and the static block fixed it. But am unable to reproduce it now 🤔

Reply:
We're kind of doing the version setup backwards, but the way I read the code the version string should be set at the second line in WynntilsMod.init(), and the WebManager not inited until later in that method. I don't know if we have some racy thing calling methods on the WebManager before that, causing it to be loaded too early.

Reply:
In any case, the static block would not fix anything, so it was probably just pure luck, if there is a race.